src/keys.rs

```diff
@@ -112,8 +115,7 @@ impl Drop for SecretKey {
 }
 
 /// A BLS public key.
-#[derive(Clone, PartialEq, Eq)]
-#[cfg_attr(feature = "std", derive(Debug))]
+#[derive(Debug, Default, Copy, Clone, PartialEq, Eq, Encode, Decode, TypeInfo, MaxEncodedLen)]
```

Comment:
Should derive RuntimeDebug instead of Debug to reduce code size in wasm.

Reply:
https://github.com/Snowfork/milagro_bls/pull/3/commits/a8e9585ae904112328ddff9c229db473b563f4fa

Reply:
Sorry can we use `#[cfg_attr(feature = "std", derive(Debug))]` instead of RuntimeDebug